pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/cdceval"
 	"github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeedbase"
+	checkpoint2 "github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/checkpoint"
```

Comment:
Instead of renaming the package, could you rename the local variable with the conflicting name? e.g. rename `checkpoint` to `spanLevelCheckpoint`. In general, you should only rename a package if you are importing two packages with the same name.